lib/apivore/version.rb

```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
+
 module Apivore
-  VERSION = "1.6.2.vsp"
+  # Forked from https://rubygems.org/gems/apivore at 1.6.2
+  VERSION = '1.6.2.1.vsp'
```

Comment:
NOTE: Using `1.6.2.1.vsp` to denote that it's the first update we've made to `1.6.2.vsp` after the initial fork of 1.6.2